pages/nurses/[id]/schedule.js

```diff
@@ -0,0 +1,99 @@
+import WorkScheduleForm from "@/components/WorkScheduleForm/WorkScheduleForm";
+import useSWR from "swr";
+import { useRouter } from "next/router";
+import { useState, useEffect } from "react";
+import fetcher from "@/utilities/fetcher";
+import { ReturnButton } from "@/components/WorkScheduleForm/WorkScheduleForm.styled";
+import WorkDatesDisplay from "@/components/WorkDatesDisplay/WorkDatesDisplay";
+import { WorkDatesContainer } from "@/components/WorkDatesDisplay/WorkDatesDisplay.styled";
+import Notification from "@/components/Notifications/Notification";
+
+export default function SchedulePage() {
+  const router = useRouter();
+  const { id } = router.query; //Nurse id
+  const { data: nurseData, mutate } = useSWR(id ? `/api/nurses/${id}` : null);
+  const { data: workDatesData } = useSWR(
+    id ? `/api/work-dates/${id}` : null,
+    fetcher
```

Comment:
you already provide this fetcher in your config, no need to pass it here a second time.